resources/lang/en-US/admin/licenses/table.php

```diff
@@ -4,6 +4,7 @@
 
     'assigned_to'   	=> 'Assigned To',
     'checkout'   		=> 'In/Out',
+    'deleted_at'        => 'Deleted at',
```

Comment:
These strings all exist in other files - we don't need to duplicate them here.

Reply:
the created by and updated at were duplicates, they have been removed. I do see ->deleted_at a lot but we dont have a translation string for it yet.